### ISSUE TITLE
.NET 6.0 does not support OLEDB

### ISSUE THREAD

**Issue by vsfeedback** [dotnet/runtime#58795]
_This issue has been moved from [a ticket on Developer Community](https://developercommunity.visualstudio.com/t/NET-60-does-not-support-OLEDB/1508889)._

---
This is the 3rd time I am trying to report a simple issue.
Please use the zip file attached, which is the solution.
1-As it is, the project is set to .NET 6.0, run the project and select the Book1.xlsx Excel file and see the result.
2-Switch to .NET 5.0 (right click solution name/property and select target framework as 5.0

run the project and see the result.

Thanks

---
#### Feedback Bot on 8/20/2021, 03:39 AM: 

<p>We have directed your feedback to the appropriate engineering team for further evaluation. The team will review the feedback and notify you about the next steps.</p>

---
### Original Solutions
(no solutions)

**Comment by ajcvickers** [comment]
@vsfeedback I downloaded your zip and ran the project with both:

```xml
<Project Sdk="Microsoft.NET.Sdk">

  <PropertyGroup>
    <OutputType>WinExe</OutputType>
    <TargetFramework>net6.0-windows7.0</TargetFramework>
    <Nullable>enable</Nullable>
    <UseWindowsForms>true</UseWindowsForms>
    <Platforms>AnyCPU;x86</Platforms>
  </PropertyGroup>

  <ItemGroup>
    <PackageReference Include="System.Data.OleDb" Version="6.0.0-preview.7.21377.19" />
  </ItemGroup>

</Project>
```

and 

```xml
<Project Sdk="Microsoft.NET.Sdk">

  <PropertyGroup>
    <OutputType>WinExe</OutputType>
    <TargetFramework>net5.0-windows7.0</TargetFramework>
    <Nullable>enable</Nullable>
    <UseWindowsForms>true</UseWindowsForms>
    <Platforms>AnyCPU;x86</Platforms>
  </PropertyGroup>

  <ItemGroup>
    <PackageReference Include="System.Data.OleDb" Version="5.0.0" />
  </ItemGroup>

</Project>
```

In both cases it behaved the same way; that is, several message boxes were popped up containing the data. For example:

![image](https://user-images.githubusercontent.com/1430078/132503850-908f4df4-e717-470b-8d6e-67427a1d9f0a.png)

Can you provide more details on exactly what is not working for you?

**Comment by ajcvickers** [comment]
No response; closing.

**Comment by Yamaha3775** [comment]
I raised the issue via Feedback in Visual Studio Preview 3.1.
I wanted to open and read an Excel file using OLEDB in a simple C# program, and I got stuck when using NET 6.0.
The issue is fixed in Preview 4.0.
Thank you Microsoft Team
